source/MapPanel.cpp

```diff
@@ -1278,11 +1278,13 @@ void MapPanel::DrawEscorts()
 				// Stored outfits are drawn/indicated by 8 short rays out of the system center.
 				for(int i = 0; i < 8; ++i)
 				{
+					constexpr float INDICATOR_LINE_RADIUS = 1.6f;
```

Comment:
This should be static.
Also, what quantity does this represent?

Reply:
> Also, what quantity does this represent?

It represents the radius/width of the indicator line. Rounded lines have cap that extends out from the endpoints by this value, so we compensate for it.

Reply:
This is not a good name; the radius of a line is not a well defined concept.
It probably shouldn't say "INDICATOR", either. This loop body is for drawing the stored outfit indicator, that's already established above, the locals scoped within that don't need to repeat that.
I think just `WIDTH` would be much better. Maybe `CAP_RADIUS`, but this doesn't really clarify that this value is also the thickness of the line.

Reply:
The static has no effect here as the constexpr is evaluated at compile time anyways.
It doesn't hurt but it also doesn't really do anything

Reply:
> The static has no effect here as the constexpr is evaluated at compile time anyways. It doesn't hurt but it also doesn't really do anything

Yeah, that's why I left it out in the first place. Oh well, consistency is an admirable goal.

Reply:
> The static has no effect here as the constexpr is evaluated at compile time anyways. It doesn't hurt but it also doesn't really do anything

`constexpr` does not imply `static`.
Without `static` the compiler will still happily run the same instruction to place this value into the stack on every iteration of this loop.

Also, `static` should come before `constexpr`.

Reply:
> `constexpr` does not imply `static`. Without `static` the compiler will still happily run the same instruction to place this value into the stack on every iteration of this loop.

In this particular case it doesn't matter, as the compiler will optimize the variable out anyways.

Reply:
Also, looking at other constexpr constants jn the codebase, most don't even have the static qualifier. It matters even less here to the point where it's useless.